cmd/vcert/main_test.go

```diff
@@ -919,3 +919,33 @@ func TestValidatePrecedenceForFlagsCloud(t *testing.T) {
 	unsetEnvironmentVariables()
 	unsetFlags()
 }
+
+func TestValidateValidDaysFlagWithValidValues(t *testing.T) {
```

Comment:
Just a minor note here. We can call this method TestValidateValidDaysFlag. I would assume that this test is for the happy path and the "whithValidValues" is redundant. Specially when the next test specifies that is using invalid values "TestValidateValidDaysFlagWithInvalidValidValues" (which is good convention).

No need to change the name if not required.

Reply:
fixed on most recent commit